src/index.js

```diff
@@ -89,6 +89,18 @@ VERBS.concat("any").forEach(function (method) {
       return _this;
     }
 
+    function replyWithDelay(delay, code, response, headers) {
+      var handler = [matcher, body, requestHeaders, code, response, headers, false, delay];
+      addHandler(method, _this.handlers, handler);
+      return _this;
+    }
+
+    function withDelayInMs(delay){
```

Comment:
Naming inconstency: above method `replyWithDelay` does not have `InMs` suffix.

Rename to `withDelay`?